client/src/shared/components/Navbar/index.tsx

```diff
@@ -34,37 +39,49 @@ const Navbar: React.FC<NavbarProps> = ({ navItems, dropdownItems, isSticky }) =>
                 {navItems.map((navItem) => {
                   return (
                     <div key={navItem.url}>
-                      {navItem.dropdownItems.length > 0 ? (
+                      {navItem.dropdownItems.length > 0
+                        ? (
                         <Dropdown
                           options={navItem.dropdownItems}
                           label={navItem.text}
                         />
-                      ) : (
+                          )
+                        : (
                         <NavLink
                           href={navItem.url}
                           text={navItem.text}
                         ></NavLink>
-                      )}
+                          )}
                     </div>
                   );
                 })}
               </div>
             </div>
           </div>
-          <div className="flex items-center">
-            <Link href={'/settings'} className="text-gray-600">
-              <SettingsIcon width={20} height={20} className="mr-2" />
-            </Link>
-            <Avatar name="Jane Doe" />
-            <Dropdown options={dropdownItems} label="John Doe" />
-          </div>
+          {signedIn
```

Comment:
no need to use useStatae you can use directly the function

```suggestion
          {isSignedIn() ? (
```